toolchain/driver/driver.cpp

```diff
@@ -236,6 +241,12 @@ auto Driver::RunDumpSubcommand(DiagnosticConsumer& consumer,
                   /*IsForDebug=*/true);
   }
 
+  if (dump_mode == DumpMode::ObjectCode) {
+    CodeGen code_gen = CodeGen();
+    code_gen.GenerateObjCodeFromModule(*module);
```

Comment:
What's your plan for how this'll look when actually generating, versus printing as you do here? Should this have a different function name?